### DISCUSSION TITLE
Issue upgrading from 9.3: Loading secrets from AWS SSM

### DISCUSSION THREAD

**Discussion by hgezim** [vercel/next.js#26465] [Help]
I'm deploying to AWS ECS and part of the setup is the environment gets loaded with env variables that are secrets from AWS SSM.

It seems like since 9.4, Next.js only supports loading variables from a .env* file instead of directly from the system environment variables at build time.

How do you folks handle creating secrets and loading them using this .env* file method?

**Comment by timneutkens** [comment]
> It seems like since 9.4, Next.js only supports loading variables from a .env* file instead of directly from the system environment variables at build time.

If you have environment variables they're always available.
